antenna/configlib.py

```diff
@@ -0,0 +1,332 @@
+"""This module contains the configuration infrastructure allowing for deriving
+configuration from a .ini file and the process environment.
+
+In order of precedence:
+
+1. settings overrides (usually only applies when running tests)
+2. process environment
+3. .ini file (specified by ANTENNA_INI environment variable)
+
+Example of usage::
+
+    from antenna.configlib import config
+
+    DEBUG = config('DEBUG', default='True', parser=bool)
```

Comment:
I still don't understand what point of this is. Sorry. Being able to use `config('DEBUG', default=True)` feels a lot more pythonic. The parser can be confidently guessed by the type of the default value. 

And parser isn't just `bool` anyway. It needs to be that fancy function that can recognize 'yes', '1', etc. 

Also, it just feels like typing `'True'` is icky.